src/common/guac_surface.c

```diff
@@ -1054,13 +1347,23 @@ void guac_common_surface_flush(guac_common_surface* surface) {
 
             /* Re-add to queue if there's room and this update was modified or we expect others might be */
             if ((combined > 1 || i < original_queue_length)
-                    && surface->png_queue_length < GUAC_COMMON_SURFACE_QUEUE_SIZE)
+                    && surface->bitmap_queue_length < GUAC_COMMON_SURFACE_QUEUE_SIZE)
                 __guac_common_surface_flush_to_queue(surface);
 
-            /* Flush as PNG otherwise */
-            else {
-                if (surface->dirty) flushed++;
-                __guac_common_surface_flush_to_png(surface);
+            /* Flush as bitmap otherwise */
```

Comment:
Yes. The flush is no longer specific to any one format of image, hence the rename from PNG-specific comments and structure to, simply, "bitmap".